src/firestore/indexes.ts

```diff
@@ -31,15 +36,59 @@ export class FirestoreIndexes {
     const indexesToDeploy: Spec.Index[] = spec.indexes;
     const fieldOverridesToDeploy: Spec.FieldOverride[] = spec.fieldOverrides;
 
-    const existingIndexes = await this.listIndexes(project);
-    const existingFieldOverrides = await this.listFieldOverrides(project);
+    const existingIndexes: API.Index[] = await this.listIndexes(options.project);
```

Comment:
I'm always confused why types are needed in these cases? Doesn't TS bring this in from the return annotation on `listIndexes`? (I spy them elsewhere too, so I see "consistency" coming)

Reply:
To me it makes the code much more readable since in this code there are two `Index` types (`Spec.*` and `API.*`) and same with `Field`.  But no, it's not required.